### PR TITLE
including extension for glmmTMB objects

### PR DESCRIPTION
**Pull request by graiffgarcia**
First of all, I absolutely love texreg. Thank you so much for the great work! The extract.glmmTMB function in my PR is a lightly modified version of extract.zeroinfl. I haven't tested it extensively, but it worked just fine on all the glmmTMB mods I have in my .GlobalEnv.

### CODE REVIEW

File: R/extract.R [hunk 1]
```diff
@@ -2273,6 +2273,101 @@ setMethod("extract", signature = className("gamlss", "gamlss"),
...
+# -- extract.gamlssZadj (gamlss.inf) -------------------------------------------
+
+#' @noRd
+extract.gamlssZadj <- function(model, type = c("qr", "vcov"), robust = FALSE,
+                               include.nobs = TRUE, include.gaic = TRUE, ...) {
+
+  type <- match.arg(type)
+  # VCOV extraction; create coefficient block
+  if (type == "vcov") {
+    covmat <- suppressWarnings(stats::vcov(model, type = "all", robust = robust,
```
Comment (leifeld) [inline R/extract.R:2285]: When `robust = TRUE` and `type = "vcov"`, the `vcov` call produces an error, at least with the default help page example of `gamlssZadj`. Can you check and fix? See also the last line in the unit test to produce the error.

File: tests/testthat/test-extract.R [hunk 2]
```diff
@@ -137,9 +135,38 @@ test_that("extract felm objects from the lfe package", {
...
+  expect_length(tr@se, 3)
+  expect_length(tr@pvalues, 3)
+  expect_length(tr@ci.low, 0)
+  expect_length(tr@ci.up, 0)
+  expect_equivalent(which(tr@gof.decimal), 2)
+  expect_equivalent(tr@coef.names, c("$\\mu$ (Intercept)",
+                                     "$\\sigma$ (Intercept)",
+                                     "$\\nu$ (Intercept)"))
+
+  tr <- extract(t0, robust = TRUE, type = "vcov")
```
Comment (leifeld) [inline tests/testthat/test-extract.R:165]: This throws an error, as per my comment above.

**Comment by graiffgarcia** [comment]
Hi Philip. Thank you for the review! 

You're right, of course; type="vcov" with robust=TRUE throws an error. The origin of the error is the gamlss:::vcov.gamlss() function; it happens with any model that uses a one-parameter distribution — which occurs in gamlssZadj, as the zero model uses a binomial distribution. Since robust = TRUE does nothing when type = "qr", my solution is the same as what you proposed: I removed the robust argument from extract.gamlssZadj and any references to it in the body of the function.

**Comment by leifeld** [comment]
Thanks for your quick response and, again, for your contribution. I will merge this into master now.

**Pull request merged**